### ISSUE TITLE
Outdated baseline. Class comments are not rendering

### ISSUE THREAD

**Issue by olekscode** [pillar-markup/MicroDown#41]
When I create a new Pharo 9 image and load the baseline of Microdown, the class comments do not render. However, if I also install the baseline of Pillar, everything is OK. So it seems that some necessary packages of Pillar are igrored by Microdown's baseline.

**Comment by olekscode** [comment]
If I install Microdown, class comments are not rendering.
<img width="1440" alt="Screenshot_2020-07-06_at_17 43 02" src="https://user-images.githubusercontent.com/8271691/86740640-1833be80-c037-11ea-9403-5d3ea10e451d.png">

I described this problem in issue #41 

I received error messages that `PRPillarParser` is missing. And since it is part of the `'parser'` group, I added that group to the baseline. After that everything was fine.

<img width="1440" alt="Screenshot 2020-07-07 at 00 35 26" src="https://user-images.githubusercontent.com/8271691/86740268-cee36f00-c036-11ea-871b-5cb62f02802f.png">

Adding `'parser'` to the baseline seemed to have fixed the problem.
But I don't know if this is the best solution.
